app/models/backend_api_config.rb

```diff
@@ -27,8 +27,7 @@ class BackendApiConfig < ApplicationRecord
   scope :by_backend_api, ->(backend_api_id) { where.has { self.backend_api_id == backend_api_id } }
 
   scope :with_subpath, -> do
-    common_query = where.not(path: '/')
-    System::Database.oracle? ? common_query.where('path is NOT NULL') : common_query.where.not(path: '')
+    where.not(path: ConfigPath::EMPTY_PATH)
```

Comment:
This is great, but we can also simplify its test accordingly. So we should just remove the next 2 lines:
https://github.com/3scale/porta/blob/fedd08b63974d66bece271d608704a8a053674b4/test/models/backend_api_config_test.rb#L30
https://github.com/3scale/porta/blob/fedd08b63974d66bece271d608704a8a053674b4/test/models/backend_api_config_test.rb#L37

Reply:
Btw this scope now can be done in 1 line:
```ruby
scope :with_subpath, -> { where.not(path: ConfigPath::EMPTY_PATH) }
```